configDesktop.js

```diff
@@ -78,10 +78,7 @@ const tests = [
 	},
 	{
 		label: '1 column (#vector-2022, #logged-in, #toc-unpinned, #main-menu-unpinned, #page-tools-unpinned)',
-		path: '/wiki/Test',
-		viewports: [
```

Comment:
This shouldnt be restricted to just the desktop widest viewport, was a mistake from a previous PR